api/v1beta1/tobiko_types.go

```diff
@@ -220,12 +221,15 @@ type TobikoWorkflowSpec struct {
         // Container image for tobiko
         ContainerImage string `json:"containerImage,omitempty"`
 
-        // BackoffLimimt allows to define the maximum number of retried executions (defaults to 6).
+        // BackoffLimit allows to define the maximum number of retried executions (defaults to 0).
+        // +kubebuilder:validation:Optional
+        // +kubebuilder:default:=0
         // +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:number"}
-        BackoffLimit *int32 `json:"backoffLimit,omitempty"`
+        BackoffLimit *int32 `json:"backoffLimit"`
```

Comment:
Parameters in workflow section should not have default value. This section should stay the same. Except for:
```
// BackoffLimit allows to define the maximum number of retried executions (defaults to 0).
```